cc/README.md

```diff
@@ -54,15 +45,15 @@ enabled for a particular board size. Consequently, you must run the tests
 multiple times:
 
 ```shell
-bazel test --define=board_size=9 cc/...  &&  bazel test --define=board_size=19 cc/...
+bazel test --define=board_size=9 cc/...  &&  bazel test cc/...
```

Comment:
do we need to update the test dockerfiles?

Reply:
The tests will continue to work as the --define=board_size=19 will just be ignored. The test dockerfiles should probably be updated to avoid confusion in the future though. I'll create a new issue.